src/main/flight/pid.h

```diff
@@ -190,6 +190,7 @@ typedef struct pidProfile_s {
     uint8_t ff_smooth_factor;               // Amount of smoothing for interpolated FF steps
     uint8_t dyn_lpf_curve_expo;             // set the curve for dynamic dterm lowpass filter
     uint8_t level_race_mode;                // NFE race mode - when true pitch setpoint calcualtion is gyro based in level mode
+    uint8_t vbat_sag_compensation;          // Reduce motor output by this percentage of the maximum compensation amount
```

Comment:
Any time you change the `pidProfile_t` structure you need to increment the PG version otherwise it will lead to settings corruption. In pid.c:
```
PG_REGISTER_ARRAY_WITH_RESET_FN(pidProfile_t, PID_PROFILE_COUNT, pidProfiles, PG_PID_PROFILE, 14);
```

For most PG's simply adding to the end is ok. But the `pidProfile` PG is an array of profiles so even adding to the end requires a version update (because it shifts the other elements of the array around).

Reply:
Thanks for reminding me of this.  Incremented now.